gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/util/ResourceReferenceUtil.java

```diff
@@ -152,28 +158,66 @@ private static void addMedicationEntry(RCMRMT030101UK04MedicationStatement medic
         }
     }
 
-    private static void addBloodPressureEntry(RCMRMT030101UK04CompoundStatement compoundStatement, List<Reference> entryReferences) {
-        entryReferences.add(createResourceReference(ResourceType.Observation.name(), compoundStatement.getId().get(0).getRoot()));
+    private static void addBloodPressureEntry(RCMRMT030101UK04CompoundStatement compoundStatement,
+                                              List<Reference> entryReferences) {
+        entryReferences.add(createResourceReference(ResourceType.Observation.name(),
+                compoundStatement.getId().get(0).getRoot()));
     }
 
-    private static void addImmunizationEntry(RCMRMT030101UK04ObservationStatement observationStatement, List<Reference> entryReferences) {
-        entryReferences.add(createResourceReference(ResourceType.Immunization.name(), observationStatement.getId().getRoot()));
+    private static void addImmunizationEntry(RCMRMT030101UK04ObservationStatement observationStatement,
+                                             List<Reference> entryReferences) {
+        entryReferences.add(createResourceReference(ResourceType.Immunization.name(),
+                observationStatement.getId().getRoot()));
     }
 
     private static void addUncategorisedObservationEntry(RCMRMT030101UK04ObservationStatement observationStatement,
         List<Reference> entryReferences) {
-        entryReferences.add(createResourceReference(ResourceType.Observation.name(), observationStatement.getId().getRoot()));
+        entryReferences.add(createResourceReference(ResourceType.Observation.name(),
+                observationStatement.getId().getRoot()));
     }
 
-    private static void addPlanStatementEntry(RCMRMT030101UK04PlanStatement planStatement, List<Reference> entryReferences) {
+    private static void addPlanStatementEntry(RCMRMT030101UK04PlanStatement planStatement,
```

Comment:
Looks like some formatting has changed, any chance we can revert it to make the review cleaner or is the code more readable with the new format?